Swift/Tests/CBLTestCase.swift

```diff
@@ -80,6 +80,8 @@ class CBLTestCase: XCTestCase {
     override func setUp() {
         super.setUp()
         
+        LogSinks._resetApiVersion()
```

Comment:
I would have this at the end of setUp, same as Obj-C, just because any logging after this might crash tests if the tests that are run are using a different API version vs any "debug" logging made in setUp. I believe it is the most fail-safe we can do.